websites/docs-smoke-test/src/content/index.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Explore how to build a documentation website for commercetools
 ---
+import { IsUserLoggedIn, IsUserLoggedOut } from '@commercetools-docs/gatsby-theme-learning'
```

Comment:
I think this component is small enough to be provided automatically through the shortcodes file.

Reply:
Sure, makes sense